src/Middleware/Zed/Process/Business/Mapper/Map/ProductImportMap.php

```diff
@@ -19,13 +20,24 @@ public function __construct(string $preGeneratedMapPath = '')
     {
         $this->preGeneratedMapPath = $preGeneratedMapPath;
     }
+
+    /**
+     * @return \Generated\Shared\Transfer\MapperConfigTransfer
+     */
+    public function getMapperConfig(): MapperConfigTransfer
+    {
+        $mapperConfigTransfer = new MapperConfigTransfer();
+        $mapperConfigTransfer->setMap($this->getMap());
+        $mapperConfigTransfer->setStrategy($this->getStrategy());
+        return $mapperConfigTransfer;
+    }
     
     /**
      * @return array
      */
-    public function getMap(): array
+    protected function getMap(): array
     {
-        $generated_map = ($this->preGeneratedMapPath != '') ? unserialize(file_get_contents($this->preGeneratedMapPath)) : [];
+        $generated_map = ($this->preGeneratedMapPath != '') ? json_decode(file_get_contents($this->preGeneratedMapPath), true) : [];
```

Comment:
We might want to think about providing a standard method for this from `SprykerMiddleware` namespace.